src/DashboardUI/src/data-contracts/RegulationDetails.ts

```diff
@@ -0,0 +1,17 @@
+// TODO: Update with finalized attribute names
+export interface RegulationDetails {
```

Comment:
```suggestion
export interface RegulatoryDetails {
```
As well as the file name to match.